core/src/main/java/org/geysermc/geyser/entity/type/player/SessionPlayerEntity.java

```diff
@@ -425,6 +427,28 @@ public void setVehicleJumpStrength(int vehicleJumpStrength) {
         this.vehicleJumpStrength = MathUtils.constrain(vehicleJumpStrength, 0, 100);
     }
 
+    @Override
+    public void setVehicle(Entity entity) {
+        // For boat, we send width 0.6 and height 1.6 since there is a problem with player "clipping" into the boat
+        // and having a wide bounding box fixed that, the problem is that this is incorrect and create certain problems
```

Comment:
```suggestion
        // For boats, we send width = 0.6 and height = 1.6 since there is otherwise a problem with player "clipping" into the boat when standing on it or running into it.
        // Having a wide bounding box fixed that, however, it is technically incorrect and creates certain problems
```